src/services/refact/prompts.ts

```diff
@@ -27,7 +32,19 @@ export const promptsApi = createApi({
           credentials: "same-origin",
           redirect: "follow",
         });
-        if (result.error) return { error: result.error };
+        if (result.error) {
+          // getting first 2 lines of error message to show to user
+          const errorMessage = (
+            result.error.data as CustomizationErrorResponse
```

Comment:
There should be a type guard for `isDetailMessage` this will remove the need for type casting